src/libsystemd/sd-netlink/netlink-message-nfnl.c

```diff
@@ -12,7 +12,7 @@
 #include "netlink-types.h"
 #include "netlink-util.h"
 
-static bool nfproto_is_valid(int nfproto) {
+bool nfproto_is_valid(int nfproto) {
```

Comment:
Could you drop DECNET below? It is deprecated.

Reply:
ok, it's not used anywhere